test/libsolidity/GasCosts.cpp

```diff
@@ -74,7 +74,7 @@ BOOST_AUTO_TEST_CASE(string_storage)
 	else if (Options::get().useABIEncoderV2)
 		CHECK_GAS(151819, 142753, 100);
```

Comment:
This one is still failing. 
```suggestion
		CHECK_GAS(151819, 136003, 100);
```